### DISCUSSION TITLE
"Any 1.x.x snapshot" filter

### DISCUSSION THREAD

**Discussion by Madis0** [modrinth/.github#17] [Feature requests]
### Is your feature request related to a problem? Please describe.

Currently the snapshots are all separated, so they assume the author assigns their mods only the appropriate ones. However, as there are so many snapshots, this is not always the case. This is bad because at the later stage of the development, there is a higher chance of a snapshot mod working with later snapshots or even the release version.

### Describe the solution you'd like

Therefore it would be great if we could filter by any 1.x.x snapshot, e.g. "any 1.18.2 snapshot" or "any 1.19 snapshot".

### Describe alternatives you've considered

Some kind of easier way to select ranges of snapshots...

**Comment by Minenash** [comment]
Since on the rewrite, the version selection in the search is a scrollable list instead of a dropdown, one way to do it would be to have "x-snapshots" with a "\\/" that when clicked expands to show the snapshots for that version.

This idea could also be used to add "sub-categories" and "sub-loaders"

**Comment by triphora** [comment]
Related to #15
